modules/agar/src/main/ts/ephox/agar/assertions/ApproxStructures.ts

```diff
@@ -133,6 +135,7 @@ const element = (tag: string, fields: ElementFields): StructAssert => {
 
       assertHtml(optHtml, actual);
       assertValue(optValue, actual);
+      assertTextContent(optTextContent, actual);
       assertChildren(optChildren, actual);
```

Comment:
I missed this one. It already exists under `children`, by using `ApproxStructures.text`
https://github.com/tinymce/tinymce/blob/e9a1186a9d8397dd5c08f343135bd0bc66786004/modules/agar/src/test/ts/browser/ApproxStructureTest.ts#L139-L144

- [ ] revert (and update tests that use it)